src/scene/shader-lib/programs/lit-shader.js

```diff
@@ -125,6 +125,13 @@ class LitShader {
 
         // generated by fshader
         this.fshader = null;
+
+        /**
+         * @type {Function|undefined} - This function may update `this.fshader` in case it's
+         * set in src/deprecated/deprecated.js
+         * @ignore
+         */
+        this.handleCompatibility;
```

Comment:
```suggestion
        this.handleCompatibility = null;
```

Reply:
Sorry, this is wrong, if you assign it, it will overwrite the actual compatibility function from `deprecated.js`. The ESLint rule here is/was deceptive.

Reply:
ah, so what's the better solution?

Reply:
maybe just to remove this line completely?